src/Initialization/init_ModelingToolkit.jl

```diff
@@ -1,17 +1,18 @@
 eval(quote
-    using .ModelingToolkit: get_variables,
-                            gradient,
+    using .ModelingToolkit: gradient,
                             simplify,
-                            Operation
+                            Num, # variable like, e.g. x[1]
+                            Term # term like, eg. x[1] + x[2] == 1
 
    """
-       _vec(vars::NTuple{L, Union{<:Operation, <:Vector{Operation}}}) where {L}
+       _vec(vars::NTuple{L, Union{<:Num, <:Vector{Num}}}) where {L}
 
    Transform a tuple of operations into one vector of operations.
 
    ### Input
 
-   - `vars` -- tuple where each element is either an `Operation` or a vector of operations
+   - `vars` -- tuple where each element is either variable like (`Num`) or a
```

Comment:
```suggestion
   - `vars` -- tuple where each element is either a variable-like (`Num`) or a
```
Is that what you mean?